variants/particle_xenon/variant.cpp

```diff
@@ -27,33 +27,43 @@
 const uint32_t g_ADigitalPinMap[] =
 {
   // D0 .. D8
-  26,   // P0.26 (SDA)
-  27,   // P0.27 (SCL)
-  33,   // P1.01 (SDA1) (UART1_RTS) (SPI1_SCK) (PWM3)
-  34,   // P1.02 (SCL1) (UART1_CTS) (SPI1_MOSI) (PWM3)
-  40,   // P1.08 (UART2_TX) (SPI1_MISO) (PWM1)
-  42,   // P1.10 (UART2_RX) (PWM1)
-  43,   // P1.11 (UART2_CTS) (PWM1)
-  44,   // P1.12 (PWM0)
-  35,   // P1.03 (UART2_RTS)
+  _PINNUM(0, 26),   // P0.26 (SDA)
+  _PINNUM(0, 27),   // P0.27 (SCL)
+  _PINNUM(1, 1),    // P1.01 (SDA1) (UART1_RTS) (SPI1_SCK) (PWM3)
+  _PINNUM(1, 2),    // P1.02 (SCL1) (UART1_CTS) (SPI1_MOSI) (PWM3)
+  _PINNUM(1, 8),    // P1.08 (UART2_TX) (SPI1_MISO) (PWM1)
+  _PINNUM(1, 10),   // P1.10 (UART2_RX) (PWM1)
+  _PINNUM(1, 26),   // P1.11 (UART2_CTS) (PWM1)
+  _PINNUM(1, 26),   // P1.12 (PWM0)
+  _PINNUM(1, 26),   // P1.03 (UART2_RTS)
```

Comment:
These are incorrectly defined. Should be 
```suggestion
  _PINNUM(1, 11),   // P1.11 (UART2_CTS) (PWM1)
  _PINNUM(1, 12),   // P1.12 (PWM0)
  _PINNUM(1, 3),   // P1.03 (UART2_RTS)
```
After making this change I was able to successfully get the analog_write sketch working on the blue on-board LED when specifying it as D7. It worked without the changes when using the LED_BUILTIN constant.

Reply:
Yep, I was able to address these changes (amongst other fixes) in the latest commit I pushed up. Thanks!

Reply:
They appear to still be misdefined in the latest commit. This would be D6, D7, and D8, and they are currently all defined as P1.26, instead of P1.11, P1.12, and P1.03 respectively.

Reply:
I suppose it helps if I committed those changes 😫. Pushed that up in the latest commit.